style.css

```diff
@@ -120,3 +120,439 @@
+  line-height: 24px;
+  top: 170px;
+  left: 20px;
+  color: white;
+}
+
+.card ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 5px;
+  height: 40px;
+  width: 316px;
+  position: absolute;
+  top: 280px;
+  left: 20px;
+}
+
+.card li {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  font-family: "Inter", serif;
+  height: 30px;
+  color: #fff;
+  background: #ffffff24;
+
+  /* border: 1px solid #8993a4; */
+  font-weight: 400;
+  font-size: 18px;
+}
+
+.card button {
+  position: absolute;
+  height: 48px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 382px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  gap: 10px;
+  background: #ff6b00;
+  border: 0;
+  justify-content: center;
+  font-size: 20px;
+  color: white;
+  font-weight: 600;
+}
+
+.one {
+  top: 783px;
+}
+
+.two {
+  top: 1245px;
+}
+
+.three {
+  top: 1707px;
+}
+
+.four {
+  top: 2169px;
+}
+
+.five {
+  top: 2631px;
+}
+
+.six {
+  top: 3093px;
+}
+
+.about {
+  display: flex;
+  position: absolute;
+  width: 100%;
+  height: 1901px;
+  left: 0;
+  top: 4500px;
+
+  /* Theme 3 / Neutral 1 */
+  background: #1c1a19;
+}
+
+.about h2 {
+  width: 327px;
+  position: relative;
+  height: 79px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 60px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  left: 20px;
+}
+
+.about p {
+  width: 327px;
+  height: 148px;
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  position: absolute;
+  left: 20px;
+  top: 130px;
+}
+
+.about button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 161px;
+  height: 48px;
+  background: #ff6b00;
+  position: absolute;
+  left: 20px;
+  border: none;
+  color: white;
+  font-size: 17px;
+  font-weight: 700;
+  top: 310px;
+}
+
+.about .img-about {
+  width: 159px;
+  height: 270px;
+  position: absolute;
+  top: 266px;
+  left: 216px;
+}
+
+.white-line {
+  position: absolute;
+  z-index: 1;
+  top: 558px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  align-self: center;
+  right: 0;
+  left: 25px;
+}
+
+.about h3 {
+  position: absolute;
+  color: white;
+  font-size: 33px;
+  top: 608px;
+  display: grid;
+  text-align: center;
+  right: 0;
+  left: 0;
+  font-family: "Meera Inimai", serif;
+}
+
+.languages {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 12px;
+  gap: 8px;
+  width: 186px;
+  height: 216px;
+  color: white;
+  list-style-type: none;
+  font-size: 23px;
+  font-family: "Meera Inimai", serif;
+  position: absolute;
+  text-align: center;
+  right: 0;
+  left: 77px;
+  justify-content: center;
+  top: 736px;
+}
```

Comment:
- [ ] Kindly add spaces between `li` items on each list to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219665011-186308bc-a651-4eb2-a85e-e39d582b7106.png) | ![image](https://user-images.githubusercontent.com/58642949/219665131-30efc971-9027-45de-b33d-ac00c8ec5223.png) |